lte/gateway/c/session_manager/UpfMsgManageHandler.cpp

```diff
@@ -70,4 +71,56 @@ void UpfMsgManageHandler::SetUPFNodeState(
   return;
 }
 
+/**
+ * Periodic messages about UPF session config
+ *
+ */
+void UpfMsgManageHandler::SetUPFSessionsConfig(
+    ServerContext* context, const UPFSessionConfigState* sess_config,
+    std::function<void(Status, SmContextVoid)> response_callback) {
+  auto& ses_config = *sess_config;
+  int32_t count    = 0;
+  conv_enforcer_->get_event_base().runInEventBaseThread([this, &count,
+                                                         ses_config]() {
+    for (int i = 0; i < ses_config.upf_session_state_size(); i++) {
+      // Deleting the IMSI prefix from imsi
+      std::string imsi_upf = ses_config.upf_session_state(i).subscriber_id();
+      std::string imsi     = imsi_upf.substr(4, imsi_upf.length() - 4);
```

Comment:
why do we remove the prefix here?

Reply:
I'm concerned that this may cause some issues in the future. All other types of sessions have IMSI stored with the prefix set. We should converge to one. This is out of scope for this PR, do you mind filing an issue to track this?

Reply:
@GANESH-WAVELABS

Reply:
Sure Marie I will file an issue to track this.